app/views/kudo_mailer/batch_kudo_notification.html.erb

```diff
@@ -1,10 +1,10 @@
 <% content_for :message do %>
 
   <% @commentables.each_with_index do |commentable, index| %>
-  
-    <% givers_list = @kudo_givers["#{commentable.class.name}_#{commentable.id}"] %>
-    <%= givers_list.each_with_index.map {|g,i| style_bold((g == "a guest" && i == 0) ? g.capitalize : g)}.to_sentence.html_safe %> 
-    left kudos on
+    <% givers_list = @kudo_givers["#{commentable.id}"] %>
+    <%= if ! givers_list.nil? then givers_list.each_with_index.map {|g,i| style_bold(( i == 0) ? g.capitalize : g)}.to_sentence.html_safe end  %>
```

Comment:
I'm surprised Hound didn't have anything to say about this, but `if !thing` (ie, if + negative condition) smells like an anti-pattern in a language that has an `unless`. I also don't entirely understand the changes here with regards to the "a guest" text, but I'll assume you tested it locally and it'll be okay.

Also the spaces just inside the brackets in `( i == 0)` are unbalanced and that makes me wince (remove the first space or add a space after the `0`). :)